src/PeopleSDKAdapter.js

```diff
@@ -0,0 +1,119 @@
+import {Observable, from, fromEvent, merge} from 'rxjs';
+import {PeopleAdapter, PersonStatus} from '@webex/component-adapter-interfaces';
+import {publish, refCount, flatMap, map, filter} from 'rxjs/operators';
+
+const USER_PRESENCE_UPDATE_EVENT = 'event:apheleia.subscription_update';
+
+/**
+ * The `PeopleSDKAdapter` is an implementation of the `PeopleAdapter` interface.
+ * This adapter utilizes the Webex JS SDK to fetch data about a person.
+ *
+ * @export
+ * @class PeopleSDKAdapter
+ * @extends {PeopleAdapter}
+ */
+export default class PeopleSDKAdapter extends PeopleAdapter {
+  constructor(datasource) {
+    super(datasource);
+
+    this.getPersonObservables = {};
+  }
+
+  /**
+   * Returns a proper Enum key for a given PersonStatus value
+   *
+   * @param {string} status
+   * @returns {string} PersonStatus
+   * @memberof PeopleSDKAdapter
+   */
+  getStatus(status) {
+    return Object.keys(PersonStatus).find((key) => PersonStatus[key] === status);
+  }
+
+  /**
+   * Encodes the personID into base64 format and return the UUID
+   *
+   * @param {string} ID
+   * @returns {string}
+   * @memberof PeopleSDKAdapter
+   */
+  getPersonUUID(ID) {
+    return Buffer.from(ID, 'base64')
```

Comment:
Can you use the helper method from the sdk? This uses the cisco secure requirement of the `UrlSafeBase64` decoder.

```js
import {base64, retry} from '@webex/common';

base64.decode(ID)
```

Reply:
then if that's the case, `webex` module needs to be a dependency. not a `peer` anymore.